tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -76,29 +76,29 @@ public async Task ARLINGTON_ROPC_AAD_Async()
         public async Task ARLINGTON_ROPC_ADFS_Async()
         {
             var labResponse = await LabUserHelper.GetArlingtonADFSUserAsync().ConfigureAwait(false);
-            await RunHappyPathTestAsync(labResponse).ConfigureAwait(false);
+            await RunHappyPathTestAsync(labResponse, isAad: true).ConfigureAwait(false);
```

Comment:
I don't think this "isAad" param is needed. These are all AAD tests. The only ADFS ROPC test is here: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/d1d3a6a88031c7123efa21ad6e663cafcd9efb7e/tests/Microsoft.Identity.Test.Integration.NetFx/HeadlessTests/UsernamePasswordIntegrationTests.cs#L106